os/pm/pm_idle.c

```diff
@@ -164,6 +164,8 @@ void pm_idle(void)
 		/* Send signal to shutdown other cores here */
 		for (cpu = 1; cpu < CONFIG_SMP_NCPUS; cpu++) {
 			if (up_get_cpu_state(cpu) == CPU_RUNNING) {
+				/* Ensure that secondary CPU systick PPI stops firing to prevent deadlock condition */
+				up_timer_set_hotplug_status(cpu, 1);
```

Comment:
✔ The up_timer_set_hotplug_status is never called without up_cpu_hotplug(). 
✔ So, we don't need to call sepeatliy up_timer_set_hotplug_status and up_cpu_hotplug. 

✨How about move calling up_timer_set_hotplug_status into up_cpu_hotplug()? 👀

Reply:
I don't know the other chipset yet. 😥
but the calling up_timer_set_hotplug_status is pm_idle of OS layer. 

so, if the function should be called in os layer, in up_cpu_hotplug() is better. 😊